src/styles/overwrite-tachyons.scss

```diff
@@ -34,396 +34,20 @@ code,
-}
-
-.grey-70 {
-  color: var(--grey-70);
-}
-
-.grey-90 {
-  color: var(--grey-90);
-}
+@each $c in $colors {
+  .#{$c} {
+    color: var(--#{$c});
+  }
 
-// Background colors
-
-// Blue
-
-.bg-blue-50 {
-  background-color: var(--blue-50);
-}
-
-.bg-blue-60 {
-  background-color: var(--blue-60);
-}
-
-.bg-blue-70 {
-  background-color: var(--blue-70);
-}
-
-.bg-blue-80 {
-  background-color: var(--blue-80);
-}
-
-.bg-blue-90 {
-  background-color: var(--blue-90);
-}
-
-// Teal
-
-.bg-teal-50 {
-  background-color: var(--teal-50);
-}
-
-.bg-teal-60 {
-  background-color: var(--teal-60);
-}
-
-.bg-teal-70 {
-  background-color: var(--teal-70);
-}
-
-.bg-teal-80 {
-  background-color: var(--teal-80);
-}
-
-.bg-teal-90 {
-  background-color: var(--teal-90);
-}
-
-// Magenta
-
-.bg-magenta-50 {
-  background-color: var(--magenta-50);
-}
-
-.bg-magenta-60 {
-  background-color: var(--magenta-60);
-}
-
-.bg-magenta-70 {
-  background-color: var(--magenta-70);
-}
-
-.bg-magenta-80 {
-  background-color: var(--magenta-80);
-}
-
-.bg-magenta-90 {
-  background-color: var(--magenta-90);
-}
-
-// Green
-
-.bg-green-50 {
-  background-color: var(--green-50);
-}
-
-.bg-green-60 {
-  background-color: var(--green-60);
-}
-
-.bg-green-70 {
-  background-color: var(--green-70);
-}
-
-.bg-green-80 {
-  background-color: var(--green-80);
-}
-
-.bg-green-90 {
-  background-color: var(--green-90);
-}
-
-// Yellow
-
-.bg-yellow-50 {
-  background-color: var(--yellow-50);
-}
-
-.bg-yellow-60 {
-  background-color: var(--yellow-60);
-}
-
-.bg-yellow-70 {
-  background-color: var(--yellow-70);
-}
-
-.bg-yellow-80 {
-  background-color: var(--yellow-80);
-}
-
-.bg-yellow-90 {
-  background-color: var(--yellow-90);
-}
-
-// Red
-
-.bg-red-50 {
-  background-color: var(--red-50);
-}
-
-.bg-red-60 {
-  background-color: var(--red-60);
-}
-
-.bg-red-70 {
-  background-color: var(--red-70);
-}
-
-.bg-red-80 {
-  background-color: var(--red-80);
-}
-
-.bg-red-90 {
-  background-color: var(--red-90);
-}
-
-// Purple
-
-.bg-purple-50 {
-  background-color: var(--purple-50);
-}
-
-.bg-purple-60 {
-  background-color: var(--purple-60);
-}
-
-.bg-purple-70 {
-  background-color: var(--purple-70);
-}
-
-.bg-purple-80 {
-  background-color: var(--purple-80);
-}
-
-.bg-purple-90 {
-  background-color: var(--purple-90);
-}
-
-// Ink
-
-.bg-ink-70 {
-  background-color: var(--ink-70);
-}
-
-.bg-ink-80 {
-  background-color: var(--ink-80);
-}
-
-.bg-ink-90 {
-  background-color: var(--ink-90);
-}
-
-// Grey
-
-.bg-grey-10 {
-  background-color: var(--grey-10);
-}
-
-.bg-grey-30 {
-  background-color: var(--grey-30);
-}
-
-.bg-grey-50 {
-  background-color: var(--grey-50);
-}
-
-.bg-grey-70 {
-  background-color: var(--grey-70);
-}
-
-.bg-grey-90 {
-  background-color: var(--grey-90);
-}
-
-// Border colors
-
-.b--green-60 {
-  border-color: var(--green-60);
-}
+  .bg-#{$c} {
+    background-color: var(--#{$c});
+  }
 
-.b--red-60 {
-  border-color: var(--red-60);
+  .b--#{$c} {
```

Comment:
We only want red and green borders?  I think we could just hardcode those.  Or have sub-lists…